package.json

```diff
@@ -87,7 +87,7 @@
       }
     }
   },
-  "devDependencies": {
+  "dependencies": {
```

Comment:
Not sure about this one too hahah

Reply:
We should not have devDependencies in the root package.json